CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.23.2 - 2018-08-07
+## Fixed
+- Fixed bug in 0.23.1 where importing Dash components with no props would result in an error. (Fixes [#321](https://github.com/plotly/dash/issues/321)).
+- Fixed bug in 0.23.1 where importing components with arguments that are python keywords could cause an error.
```

Comment:
Could you add a note here like "In particular, this fixes importing the modules with Python 3.7"?